src/ignitus-JobConfirmation/Components/JobConfirmation.js

```diff
@@ -0,0 +1,184 @@
+                    </p>
+                      <p>We can't wait to meet you.</p>
+                    </div>
+                    <h4 className="note">Please note that all the applications must be filled by form.</h4>
+                  </div>
+                </div>
+                <div className="job-joining-details">
+                  <div className="job-info">
+                    <h4 className="job-info-heading">
+                      Job Info
+                    </h4>
+                    <div>
+                      <h5>
+                        Location
+                  </h5>
+                      <p>
+                        Pitsburg , New Jersy
+                  </p>
+                    </div>
+                    <div>
+                      <h5>
+                        Contract
+                  </h5>
+                      <p>
+                        Internship
+                  </p>
+                    </div>
+                    <div>
+                      <h5>
+                        Job Category
+                  </h5>
+                      <p>
+                        Designer
+                  </p>
+                    </div>
+                    <div>
+                      <h5>
+                        Experience Level
+                  </h5>
+                      <p>
+                        Beginner, Intermediate, Proficient
+                  </p>
+                    </div>
+                  </div>
+                  <div className="job-contact">
+                    <h4 className="job-info-heading">
+                      Contact
+                    </h4>
+                    <div>
+                      <h5>
+                        Recruiter
+                    </h5>
+                      <p>
+                        Name
+                  </p>
+                    </div>
+                    <div>
+                      <h5>
+                        Email
+                    </h5>
+                      <p>
+                        abc@xyz.com
+                  </p>
+                    </div>
+                  </div>
+
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        {this.state.apply &&
+          <div className="card-outer">
+            <div className="card-body">
+              <div className="card-content">
+                <div className="cancelBtn"><i onClick={() => this.setState({ apply: false })} className="fa fa-times"></i></div>
+                <h3>Thank you for your application!</h3>
+                <p>Its been great to see that you are interest in joining Ignitus.Your application has been received</p>
+                <div className="card-links">
+                  <div>
+                    <div className="social-link fb">
+                      <i className="fab fa-facebook-f"></i>
```

Comment:
Please update `<i className="fab fa-facebook-f"></i>` to `<i className="fa fa-facebook-f"></i>`